specs/experimental/cannon-fault-proof-vm-mt.md

```diff
@@ -205,11 +215,13 @@ the FPVM preempts the active thread.
 
 ## Stateful Instructions
 
+### Load Linked / Store Conditional Word
+
 The Load Linked Word (`ll`) and Store Conditional Word (`sc`) instructions provide the low-level
 primitives used to implement atomic read-modify-write (RMW) operations.  A typical RMW sequence might play out as
 follows:
 
-- `ll` place a "reservation" on a particular memory address.
+- `ll` places a "reservation" on a particular memory address.
 - Subsequent instructions take the value at this address and perform some operation on it:
   - For example, maybe a counter variable is reserved and incremented.
 - `sc` is called and the modified value is stored at the reserved address only if it has not been modified since the
```

Comment:
I forget what the behavior was pre-64, but this isn't exactly correct now.
`sc` succeeds only if the word at the reserved _effective_ address has been modified. This is stricter than what's currently in the spec as a single byte modified at that word but located at a different reserved address will invalidate the reservation. So I suggest rewording this.

It probably helps to use be explicit on the type of address referenced here (virtual/effective).

Reply:
tweaked this a bit - let me know if you think it makes sense now.